CMakeLists.txt

```diff
@@ -107,10 +107,16 @@ PKG_CHECK_MODULES(SDL2 REQUIRED sdl2)
 SET(SDL2LIBS ${SDL2_DEFINITIONS})
 
 if (USE_BOOST_FILESYSTEM)
-    set(Boost_USE_STATIC_LIBS ON)
+    set(Boost_USE_STATIC_LIBS OFF)
```

Comment:
`OFF` is the default value.

Reply:
Yes, I thought so, but I would like to still force it in case some system ships with different value and we end up checking static twice rather than dynamic+static.